includes/class-widget.php

```diff
@@ -117,8 +131,9 @@ public function widget( $args, $instance ) {
 	 */
 	public function form( $instance ) {
 		echo sprintf( '<p><em>%s</em></p>', esc_html__( 'This widget will only display on single articles.', 'republication-tracker-tool' ) );
-		$title = ! empty( $instance['title'] ) ? $instance['title'] : '';
-		$text  = ! empty( $instance['text'] ) ? $instance['text'] : esc_html__( 'Republish our articles for free, online or in print, under a Creative Commons license.', 'republication-tracker-tool' );
+		$title  = ! empty( $instance['title'] )  ? $instance['title'] : '';
+		$text   = ! empty( $instance['text'] )   ? $instance['text'] : esc_html__( 'Republish our articles for free, online or in print, under a Creative Commons license.', 'republication-tracker-tool' );
+		$layout = ! empty( $instance['layout'] ) ? $instance['layout'] : ''
```

Comment:
<img width="366" alt="Screenshot 2025-02-13 at 12 21 41 PM" src="https://github.com/user-attachments/assets/ddd007cb-d382-46b5-ad7a-d1a594494299" />

The selector is rendered with no selection made, which throws an error if you try and save in WP Admin > Appearance > Widgets. For backwards compatibility, let's use `modal` as the default setting and have it pre-selected here if no setting has been saved yet.